style.css

```diff
@@ -417,6 +417,77 @@ a {
 
 /* Not display on mobile */
 
+/* Pop-up projec */
+.pop-up-project-section {
+  background-color: #1c1a19;
+  padding: 20px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2000;
+  display: none;
+  height: auto;
+}
+
+.popup {
+  display: block;
+}
+
```

Comment:
- [ ] Having made a great effort in your project, I notice the `mobile version` of your pop-up modal is not aligning very well with the `mobile screen size of 375px`. the first card in `My recent work` is cutting out at the side and bottom when on `mobile screen size of 375px` while the other cards are missing some `margin` at the bottom

  - Please make the image and the text in the pop-up modal aligned as shown in the Figma design
  - I noticed all your cards hold the same language (i.e `HTML`, `CSS`, and `Bootstrap`). Kindly give other languages you will be learning in `Microverse` (i.e `React`, `Ruby`, and `JavaScript`) to the card as this will showcase your progress and transitioning
 
Your Design 👇 

![aub-pop](https://user-images.githubusercontent.com/78634841/209226137-df6a508c-19ab-4a31-b4c3-bd1674f1c267.png)

Figma Design 👇 

![pp2-mob](https://user-images.githubusercontent.com/78634841/209226234-d01c2efb-dc0f-4bff-a6e7-bcbefee3bc5b.png)

  > Kindly stick to the Figma design as much as possible in order to show some consistency in your work and the ability to replicate any design given to you by a client.